policy/lib/refs_test.rego

```diff
@@ -73,6 +73,7 @@ test_slsav1_local_ref {
 
 test_git_resolver_in_slsav1_pipelinerun {
 	ref := {"spec": {"taskRef": {
+		"name": "git-clone",
```

Comment:
Yeah, see this not a valid taskRef AFAICT.

Reply:
I don't have an example git resolver, but this [1] is what I'm referring to here. I think it's right? Although, I don't see a `name` in the taskRun example here [2]

[1] https://github.com/tektoncd/pipeline/blob/main/pkg/apis/pipeline/v1/taskref_types.go#L20
[2] https://tekton.dev/docs/pipelines/git-resolver/